js/whitebit.js

```diff
@@ -1494,33 +1494,18 @@ module.exports = class whitebit extends Exchange {
         const accountsByType = this.safeValue (this.options, 'accountsByType');
         const fromAccountId = this.safeString (accountsByType, fromAccount, fromAccount);
         const toAccountId = this.safeString (accountsByType, toAccount, toAccount);
-        let type = undefined;
-        if (fromAccountId === 'main' && toAccountId === 'trade') {
-            type = 'deposit';
-        } else if (fromAccountId === 'trade' && toAccountId === 'main') {
-            type = 'withdraw';
-        }
-        if (type === undefined) {
-            throw new ExchangeError (this.id + ' transfer() only allows transfers between main account and spot account');
-        }
+        const amountString = amount.toString ();
         const request = {
             'ticker': currency['id'],
-            'method': type,
-            'amount': this.currencyToPrecision (code, amount),
+            'amount': this.currencyToPrecision (code, amountString),
+            'from': fromAccountId,
+            'to': toAccountId,
         };
         const response = await this.v4PrivatePostMainAccountTransfer (this.extend (request, params));
         //
         //    []
         //
-        const transfer = this.parseTransfer (response, currency);
-        const transferOptions = this.safeValue (this.options, 'transfer', {});
-        const fillTransferResponseFromRequest = this.safeValue (transferOptions, 'fillTransferResponseFromRequest', true);
-        if (fillTransferResponseFromRequest) {
-            transfer['amount'] = amount;
-            transfer['fromAccount'] = fromAccount;
-            transfer['toAccount'] = toAccount;
-        }
-        return transfer;
+        return this.parseTransfer (response, currency);
```

Comment:
@Dan-krm I think we can extend here the amount,fromAccount and toAccount values from the request